### PR TITLE
Us199 lizard history

### PR DESCRIPTION
**Pull request by ilathem**
- [x] Visually appealing on various screen sizes
- [x] Data is accurate to the customer's specs (entries in the recapture history should match with the selected toecode, site, array, and species code)

### CODE REVIEW

**Comment by ilathem** [comment]
Side by side comparisons:

Gateway Test
image 

San Pedro Test
image

Virgin River Test
image

File: src/forms/NewLizardEntry.jsx [hunk 1]
```diff
@@ -73,8 +75,52 @@ export default function NewLizardEntry() {
...
+            );
+            console.log('getting toe codes from test');
+            setSiteToeCodes(toeCodesSnapshot.data());
+        } catch (e) {
+            console.log('getting toe codes from live');
+            toeCodesSnapshot = await getDocsFromCache(
+                query(collection(db, 'ToeClipCodes'), where('SiteCode', '==', currentData.site))
+            );
+            setSiteToeCodes(toeCodesSnapshot.docs[0].data());
+            // console.log('retreiving toe codes from ' + currentData.site)
```
Comment (IanSkelskey) [inline src/forms/NewLizardEntry.jsx:92]: Do you need the comments still? Everything's looking good so far.

**Review by IanSkelskey — APPROVED** [review]
Other than left over comments. Looks good. I'll merge this now. We can clean those up before main if anything.

File: src/forms/NewLizardEntry.jsx (hunk 1 above)
Comment (ilathem) [reply src/forms/NewLizardEntry.jsx:92]: for testing

**Pull request merged**